frontend/src/screenControllers/PlayerScreenController/PlayerScreenController.js

```diff
@@ -44,6 +44,7 @@ export default function PlayerScreenController() {
 
     case 'lobby-closed':
     case 'error':
+    case 'showing-end-game-messages':
```

Comment:
We should probably name this something other than "showing" as it doesn't fit with the rest. Maybe "end-game" or something?